src/utils/prettifier.ts

```
@@ -0,0 +1,370 @@
+    state.copyBeginOffset = index + 1;
+
+    state.indentLevel = Math.max(0, state.indentLevel - 1);
+
+    if (!state.lineBreakPending) {
+        state.result += getStartingWhitespaceOfNewLine(state.indentLevel);
+        state.isPrettifiedStringDifferent = true;
+    }
+
+    state.lineBreakPending = false;
+
+    if ((state.regionStack[state.regionStack.length - 1] ?? {}).type === (isBraceOrParen ?
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)